eng/common/Update-Change-Log.ps1

```diff
@@ -0,0 +1,135 @@
+﻿# Note: This script will add or replace version title in change log
+param (
+  [Parameter(Mandatory = $true)]
+  [String]$Version,
+  [Parameter(Mandatory = $true)]
+  [String]$ChangeLogPath,
+  [String]$Unreleased = $True,
+  [String]$ReplaceVersion = $False
+)
+
+# Parameter description
```

Comment:
minor: I'd either put the parameter comments inline with the parameters or at the top of the script.